engine/render/renderer/platform/vulkan/Swapchain.h

```diff
@@ -167,11 +168,11 @@ class Swapchain
         return swapchainExtent;
     }
 
-    void BeginRenderPass(VkCommandBuffer commandBuffer,
+    void BeginRenderPass(Vulkan::CommandBuffer& commandBuffer,
                          uint32_t imageIndex,
                          std::initializer_list<VkClearValue> clearValues);
 
-    void EndRenderPass(VkCommandBuffer commandBuffer);
+    void EndRenderPass(Vulkan::CommandBuffer& commandBuffer);
```

Comment:
Likewise for these

Reply:
These change the CommandBuffer, so they can't be `const`.